package.json

```diff
@@ -11,7 +11,7 @@
         "node-uuid": "~1.4.0",
         "semver": "~1.1.4",
         "xml-mapping": "~1.2.0",
-        "kettle": "git://github.com/fluid-project/kettle.git#5a2e8b8fb5b198b266d6b7b7d0d9eb94185a3759"
+        "kettle": "git://github.com/fluid-project/kettle.git#27a3bf7cfd51f62e66b45bc1b7bae9f407052586"
```

Comment:
github has a long-standing bug/feature that the diff set shown is frozen at the version that it had when the pull/branch was first issued - that is, it always diffs against the revision that master had at that time rather than the one it has now. It can be very annoying for pulls which remain open for a long time. Perhaps one way to mitigate it might be to close the pull request and reopen it again if master has become too different.